src/plugin/helpers/validation.ts

```diff
@@ -0,0 +1,64 @@
+import type { FieldNameToValueMap } from "../fieldViews/helpers";
+import type { FieldSpec } from "../types/Element";
+
+type Validator = (fieldValue: unknown) => string[];
+
+export const buildValidator = (
```

Comment:
This is a _very_ minor naming suggestion, but maybe `createValidator` would be a better name here - I always find I have to parse 'build' mentally as a noun vs. verb - does this _validate_ a _build_, or does it _build_ a _validator_? Whereas 'create' is unambiguously a verb.

Reply:
Completely agree, great suggestion – done. I think we use 'build' elsewhere in this project – can I suggest raising another PR to change those names, too?